coralproject/cay
# Ask version number

## What does this PR do?

This PR implements the Version Number in the Sidebar. Please read:  [Issue 131](https://github.com/coralproject/ask/issues/131)
## How do I test this PR?
- Checkout branch `git checkout ask-version-number`
- Run Cay `npm start`
- Check the Version Number in the Sidebar

@coralproject/frontend
